include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -169,6 +169,31 @@ class CommonOperatorLoader : public ProtobufLoader {
     return llvm::Error::success();
   }
 
+  /// Loads PRELU operator, given its protobuf representation and parsed args.
+  /// Follows undirectional broadcasting described here:
+  /// https://github.com/onnx/onnx/blob/fb1a80692c1ab0bd27b1072f2e7bffacba336777/docs/Broadcasting.md
+  llvm::Error loadPRelu(const OpType &op, ArgumentDictionaryTy &dict) {
+    const std::string &opName = loadOperatorName(op);
+
+    NodeValue in;
+    ASSIGN_VALUE_OR_RETURN_ERR(in,
+                               getNodeValueOrCreateConstantByName(op.input(0)));
+
+    NodeValue slope;
+    ASSIGN_VALUE_OR_RETURN_ERR(slope,
+                               getNodeValueOrCreateConstantByName(op.input(1)));
+
+    // Do broadcasting.
+    auto targetDim = in.dims();
+    // Sets the axis of each inputs so that the trailing-most dimensions of
+    // input tensors and the target shape are aligned.
+    int axis = targetDim.size() - slope.dims().size();
+    auto *finalSlope = G_.createBroadcast(opName, slope, targetDim, axis);
```

Comment:
as talked offline, you mentioned that broadcast not always required. Let's make sure that if we insert it it's optimized away. If not then make sure we only create broadcast when needed.